### DISCUSSION TITLE
Uninstall cask without prompt asking password

### DISCUSSION THREAD

**Discussion by DiegoUY** [Homebrew/discussions#3966] [Casks] [answered]
Hi everyone,
My team and I received the task to delete an application installed with `brew install --cask`. Our intention is to resolve this with a shell script executed through our MDM solution, but we get the problem that any time we try to execute de `brew uninstall --cask` prompt us asking the user password.

We already try to pipe the password with an echo without success. Is there any other way to achieve this?

Thanks in advance and sorry if this is not the place to ask this.
Regards,

**Comment by p-linnane** [comment, marked as answer]
Have you tried `SUDO_ASKPASS`?
https://stackoverflow.com/questions/12608293/how-to-setup-a-sudo-askpass-environment-variable

**Reply by DiegoUY** [reply]
No, after your recommendation i try out and work like a charm. Thanks a lot!
